index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello Microverse</h1>
```

Comment:
- [ ] You should  add some style to this tag. [Requirement](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/hello_microverse_project.md#:~:text=A%20.css%20file%20that%20includes%20one%20class%20with%20styles%20for%20your%20header%20(make%20it%20in%20your%20favorite%20color).)

Reply:
style added in style.css as a class and implemented inside the index.html <h1> tag